code/modules/busy_space/organizations.dm

```diff
@@ -615,9 +859,47 @@
 	work = "courier and passenger transit"
 	headquarters = "Mars, Sol"
 	motto = "With Major Bill's, you won't pay major bills!"
+	annoying = 100 //oh god
```

Comment:
This seems to be the only one without a legit var. Intentional?

Reply:
heph and xion also don't have legitness. the default's rare bc a lot of current orgs are NT competators, friendly governments, or Morpheus (morpheus)